### PR TITLE
Use ol.Observable.unByKey

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: src/abstractsynchronizer.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ olcs.AbstractSynchronizer.prototype.setView_ = function(view) {
  */
 olcs.AbstractSynchronizer.prototype.setLayers_ = function(layers) {
   if (!goog.isNull(this.olLayers)) {
-    goog.array.forEach(this.olLayersListenKeys_, this.olLayers.unByKey);
+    goog.array.forEach(this.olLayersListenKeys_, ol.Observable.unByKey);
```
Comment (elemoine) [inline src/abstractsynchronizer.js]: You need to add `goog.require('ol.Observable')`.

File: src/camera.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ olcs.Camera.identityProjection = function(input, opt_output, opt_dimension) {
  */
 olcs.Camera.prototype.setView_ = function(view) {
   if (!goog.isNull(this.view_)) {
-    this.view_.unByKey(this.viewListenKey_);
+    ol.Observable.unByKey(this.viewListenKey_);
```
Comment (elemoine) [inline src/camera.js]: Same comment regarding `goog.require('ol.Observable')`.

**Comment by fredj** [comment]
`make check` should have catch this error :-(

**Comment by fredj** [comment]
Thanks for the review

**Comment by gberaudo** [comment]
@fredj, would updating the compiler catch these errors?

**Comment by fredj** [comment]
Yes, probably

**Pull request merged**